test/cypress/support/component.js

```diff
@@ -67,12 +71,21 @@ Cypress.Commands.add('mount', (component, options = {}) => {
   options.global = options.global || {};
   options.global.plugins = options.global.plugins || [];
 
+  // create router if one is not provided
+  if (!options.router) {
+    options.router = createRouter({
+      routes: routes,
+      history: createMemoryHistory(),
+    });
+  }
+
   // Register Swiper third party lib component
   register();
 
   // Add i18n plugin
   options.global.plugins.push({
     install(app) {
+      app.use(options.router);
       app.use(i18nApp);
       app.use(VueLogger, loggerOptions);
       app.use();
```

Comment:
We don't need line 91, as I 'm said in my previous comment https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/121#discussion_r1393646660

Reply:
Ah, I see. Removed.